pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1027,7 +1034,7 @@ func (t *Task) createRsyncClientPods() error {
 	// Get client for destination
 	srcClient, err := t.getSourceClient()
 	if err != nil {
-		return err
+		return liberr.Wrap(err)
```

Comment:
Why is this an exception? Other errors later in the function are returned without Wraps?

Reply:
@alaypatel07 this is certainly not an exception. I just copied this block from somewhere and changed the error randomly. I will make sure I make the change everywhere. Thanks for pointing out!